QPCSC_p.h

```diff
@@ -29,10 +29,14 @@
 #define NOMINMAX
 #include <winsock2.h>
 #include <winscard.h>
-#else
+#elif Q_OS_MAC
```

Comment:
```suggestion
#elif defined(Q_OS_MAC)
```

Reply:
Thanks for going ahead, I simply forgot to come back to this issue.